doc/man3/OSSL_DESERIALIZER.pod

```diff
@@ -0,0 +1,139 @@
+=pod
+
+=head1 NAME
```

Comment:
General comment on the all of the deserializer documentation. We need *examples* of how to use this stuff. I've just read through it all and still don't quite understand how it all fits together.

Reply:
I'd rather not take more time away from OSSL_STORE work, so if I leave a TODO comment, will that be satisfactory enough for the moment being?

Reply:
ok

Reply:
TODO comments added